app/controlplane/pkg/biz/user.go

```diff
@@ -90,15 +96,30 @@ func (uc *UserUseCase) DeleteUser(ctx context.Context, userID string) error {
 	return uc.userRepo.Delete(ctx, userUUID)
 }
 
-func (uc *UserUseCase) FindOrCreateByEmail(ctx context.Context, email string) (*User, error) {
+// FindOrCreateByEmail finds or creates a user by email. By default, it will auto-onboard the user
+// to the organizations defined in the configuration. If disableAutoOnboarding is set to true, it will
+// skip the auto-onboarding process.
+func (uc *UserUseCase) FindOrCreateByEmail(ctx context.Context, email string, disableAutoOnboarding ...bool) (*User, error) {
```

Comment:
Why `...bool`? How many `bools` are we expecting here?

Reply:
It's an optional parameter otherwise we would have to add that parameter to all occurrences.